src/App.tsx

```diff
@@ -1,24 +1,78 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { FilterStatus, Todo } from './types/Todo';
+import { showError, GetFilteredTodos } from './helpers/helpers';
+import { Error } from './components/Error';
+import { Footer } from './components/Footer';
+import { Todos } from './components/Todos';
+import { Header } from './components/Header';
 
-const USER_ID = 0;
+const USER_ID = 10881;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [searchQuery, setSearchQuery] = useState('');
```

Comment:
Not sure whether we have search query here, or newTodo ;)

Reply:
agree with search query, bur array with todos, need to be here